recipes/crunch/all/conanfile.py

```diff
@@ -0,0 +1,72 @@
+import os
+import glob
+from conans import ConanFile, tools, CMake, MSBuild
+from conans.errors import ConanInvalidConfiguration
+
+
+class CrunchConan(ConanFile):
+    name = "crunch"
+    description = "Advanced DXTc texture compression and transcoding library"
+    homepage = "https://github.com/BinomialLLC/crunch"
+    topics = ("conan", "crunch", "DXTc", "texture", "compression", "decompression", "transcoding")
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "Zlib"
+    settings = "os", "arch", "compiler", "build_type"
+    generators = "cmake"
+    exports_sources = ["CMakeLists.txt", "patches/**"]
+    options = {
+        "fPIC": [True, False],
```

Comment:
no shared support?

Reply:
No, crunch doesn't exports symbols and contains global variables (so `CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS` causes linking errors). I am currently fixing and updating in a fork (https://github.com/FrozenStormInteractive/Crunch2). Crunch2 supports macOS and shared builds. I will publish it later in the CCI.

Reply:
thanks for claryfing